src/index.js

```diff
@@ -9,6 +9,15 @@ const wrap = require("./wrap");
 
 module.exports = function convexpress(options) {
     const router = Router().use(parseBody(options.bodyParserOptions));
+    router.use((err, req, res, next) => {
```

Comment:
I'm not sure this is the best place to intercept the error, because it's global to the router, and if other handlers / middlewares emit an unrelated `SyntaxError`, then a `415 Invalid JSON` response is sent, and it might not be appropriate. Handlers defined in "convroutes" shouldn't be a problem, because they are wrapped in an error handler anyway. But custom routes and middleware would be affected.

I would suggest instead "wrapping" the `jsonMiddleware` function. Looking at its source, it seems it can either throw `SyntaxError`-s ([here](https://github.com/expressjs/body-parser/blob/master/lib/types/json.js#L83) and [here](https://github.com/expressjs/body-parser/blob/master/lib/types/json.js#L91)) or pass an error to the `next` function ([here](https://github.com/expressjs/body-parser/blob/master/lib/types/json.js#L126-L129)), so I would "intercept" those two behaviours.

(Note: actually even the [`read` function](https://github.com/expressjs/body-parser/blob/master/lib/types/json.js#L134-L139) could fail and send "ugly errors". But tbh I never encountered any such errors (even the malformed-json one is quite rare), so it's probably not a big deal to leave it as it is).